whole-tale/ngx-dashboard
# Upgrade to Angular 12

## Problem
Fixes #57

## Approach
* Migrate from Angular v9 to v10
* Migrate from Angular v10 to v11
* Migrate from Angular v11 to v12
* Update dependencies (e.g. TypeScript `4.3.5`)

## How to Test
1. Checkout this branch locally, rebuild the dashboard
2. Login to the WholeTale Dashboard
3. Run a general smoke test of the system
    * You should see that the system behaves precisely as before
    * You should not see any errors in the console (warnings are ok for now, but should disappear soon as well)
